assets/js/modules/globalNav.js

```diff
@@ -144,8 +149,11 @@ define([
      * @returns {Boolean} true if spec shoud be skipped.
```

Comment:
Correct JSDom, first param is now not only a String, but could also be an array.